### PR TITLE
Cge 107 login UI bugs

### PR DESCRIPTION
**Pull request by Wolshebnik**

### CODE REVIEW

File: src/main/resources/react/src/REST/Service/networkProvider.js [hunk 1]
```diff
@@ -13,9 +9,9 @@ const instance = axios.create( {
...
 
-export const axiosInstance = async ( method, resource, { categories = '', id = '', body } ) => {
-
-	instance.defaults.headers.common['Authorization'] = 'Bearer ' + getToken();
-	instance.defaults.headers.common['accept-language'] = 'en';
-	return await instance[method]( `/${ resource }/${ categories }/${ id }`, body );
+export const axiosInstance = async ( method, url, body = null, id = '' ) => {
+	instance.defaults.headers.common['Authorization'] = 'Bearer ' + getStorageUser( 'token' );
+	instance.defaults.headers.common['accept-language'] = getStorageLang();
+console.log(body)
```
Comment (rpkyrych) [inline src/main/resources/react/src/REST/Service/networkProvider.js:15]: Why do we need `console.log` that writes to the console sensitive data?

File: src/main/resources/react/src/components/header/nav_top/NavTop.js [hunk 2]
```diff
@@ -5,25 +5,30 @@ import SelectLang from '../../selectLang';
...
 						</Link>
 						<Link to="/" className="navbar-top-link">
 							<i className="icon-heart"/>
-							{getTranslatedText('header.goodness')}
+							{getTranslatedText('header.goodness', lang)}
 						</Link>
 					</div>
 					<div id="personalArea">
-						<Link to="/userInfo">
+						<Link to={ isAuthenticated ? '/userInfo/activity':'/logIn/' }>
```
Comment (rpkyrych) [inline src/main/resources/react/src/components/header/nav_top/NavTop.js:29]: Looks like "logIn" has misspelling here.

File: src/main/resources/react/src/components/local/ua.js [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
+		regEmail: 'Введіть ваш E-mail',
+		regLogin: 'Введіть ваш логін',
+		regPassword: 'Введіть Ваш пароль',
+		regConfirm: 'Повторіть пароль',
+		remember: 'Запам\'ятати мене',
+		noLogin: 'Чи не можете увійти?',
+		agreement: '* Я погоджуюсь з правилами використання сервісу, а також з передачею і обробкою моїх даних. Я підтверджую своє повноліття і відповідальність за розміщення оголошень.'
+	},
+	errors: {
+		logEmail: 'Введіть вірний логін або email',
```
Comment (rpkyrych) [inline src/main/resources/react/src/components/local/ua.js:16]: You should change word "вірний" into "правильний".

File: src/main/resources/react/src/components/local/ua.js [hunk 4]
```diff
@@ -1,6 +1,28 @@
...
+		logPassword: 'Невірний пароль',
```
Comment (rpkyrych) [inline src/main/resources/react/src/components/local/ua.js:17]: Update to "Неправильний пароль".

File: src/main/resources/react/src/components/local/ua.js [hunk 5]
```diff
@@ -1,6 +1,28 @@
...
+		regEmail: 'Вкажіть діючу адресу електронної пошти',
+		regNick: 'Більше 2 і менше 50 символів, без пробілів',
+		regPassword: 'Латиницею великі, маленькі і цифри від 8 до 30 символів',
```
Comment (rpkyrych) [inline src/main/resources/react/src/components/local/ua.js:20]: It looks like you missed word "букви" here.

File: src/main/resources/react/src/pages/FormRegister/routesRegister.js [hunk 6]
```diff
@@ -1,12 +1,14 @@
...
+import LogIn from './LogIn';
+import SignUp from './SignUp';
 
 export default () => {
   return (
     <Switch>
-      <Route path="/registration/" exact component={Form} />
-      <Route path="/registration/register" component={Form} />
+      <Route path="/logIn/" exact component={LogIn} />
+      <Route path="/logIn/signUp" component={SignUp} />
```
Comment (rpkyrych) [inline src/main/resources/react/src/pages/FormRegister/routesRegister.js:11]: You have misspelling into both lines that send request to "logIn". All letters must be low case.

File: src/main/resources/react/src/redux/Utils/toggleDisableButtonAuth.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+export const toggleButtonLog = ( state ) => {
+	const { logEmail, logPassword } = state;
+	return (logEmail.value !== '' && logEmail.error === '' && logPassword.value !== '' && logPassword.error === '');
+};
+
+export const toggleButtonReg = ( state ) => {
+	const { regEmail, regNick, regPassword, regConfirm, regCheckbox } = state;
+	return (regEmail.value !== '' && regEmail.error === '' && regNick.value !== '' && regNick.error === '' && regPassword.value !== '' && regPassword.error === '' && regConfirm.value !== '' && regConfirm.error === '' && regCheckbox);
```
Comment (rpkyrych) [inline src/main/resources/react/src/redux/Utils/toggleDisableButtonAuth.js:8]: Why don't you create a function that would get parameter and check it's **.value** and **.error** and return result of both those checks?
The code would be much shorter.

File: src/main/resources/react/src/redux/auth/saga/workers/workers.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+		yield put( stopFetching() );
+	}
+}
+
+export function* workerPostAuthRegister( action ) {
+	const body = action.payload;
+	try {
+		yield put( startFetching() );
+		const { data } = yield call( postAuthRegister, body );
+		console.log( data );
```
Comment (rpkyrych) [inline src/main/resources/react/src/redux/auth/saga/workers/workers.js:28]: You're logging sensitive data to console here as well.

File: src/main/resources/react/src/routes/routes.js [hunk 9]
```diff
@@ -1,16 +1,25 @@
...
+	return isAuthenticated ? (
+		<Switch>
+			<Route path="/" component={ HomePage } exact/>
+			<Route path="/userInfo/" component={ UserInfo }/>
+			<Redirect to={ '/' }/>
+		</Switch>
+	) : (
+		<Switch>
+			<Route path="/" component={ HomePage } exact/>
+			<Route path="/logIn/" component={ Auth }/>
```
Comment (rpkyrych) [inline src/main/resources/react/src/routes/routes.js:21]: You have misspelling here into "logIn". All letters must be low case.

**Review by rpkyrych — CHANGES_REQUESTED** [review]
Some of your changes invokes questions... You should check my comments.

File: src/main/resources/react/src/components/header/nav_top/NavTop.js (hunk 2 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/components/header/nav_top/NavTop.js:29]: Done

File: src/main/resources/react/src/components/local/ua.js (hunk 4 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/components/local/ua.js:17]: Done

File: src/main/resources/react/src/components/local/ua.js (hunk 3 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/components/local/ua.js:16]: Done

File: src/main/resources/react/src/components/local/ua.js (hunk 5 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/components/local/ua.js:20]: what is your suggestion?

File: src/main/resources/react/src/redux/auth/saga/workers/workers.js (hunk 8 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/redux/auth/saga/workers/workers.js:28]: Done

File: src/main/resources/react/src/routes/routes.js (hunk 9 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/routes/routes.js:21]: Done

File: src/main/resources/react/src/pages/FormRegister/routesRegister.js (hunk 6 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/pages/FormRegister/routesRegister.js:11]: Done

File: src/main/resources/react/src/redux/Utils/toggleDisableButtonAuth.js (hunk 7 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/redux/Utils/toggleDisableButtonAuth.js]: thank you

**Review by rpkyrych — APPROVED** [review]

**Review by Dariy98 — APPROVED** [review]

**Pull request merged**